codespell_lib/data/dictionary.txt

```diff
@@ -9702,6 +9702,9 @@ catgorized->categorized
 catgorizes->categorizes
 catgorizing->categorizing
 catgory->category
+cathegorising->categorising
+cathegorizing->categorizing
```

Comment:
Time permitting, could you add all verbal forms?
```
cathegorise->categorise
cathegorised->categorised
cathegorises->categorises
cathegorising->categorising
cathegorize->categorize
cathegorized->categorized
cathegorizes->categorizes
cathegorizing->categorizing
```

Reply:
Done